Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'jekyll'
```

Comment:
Yeah, your question makes sense! To be honest, I'm not sure if it's normal for Jekyll. I personally think it makes more sense to have it though, for ease of setup on new machines / for new developers. There's a bit of a precedent for this pattern / style in Rails - Gemfiles for new projects include Rails as a dependency as well.